src/Faker/Provider/ru_RU/Address.php

```diff
@@ -14,55 +14,38 @@ class Address extends \Faker\Provider\Address
 
     protected static $buildingNumber = array('##');
     protected static $postcode = array('######');
+    
+    /**
+     * @link http://www.consultant.ru/document/cons_doc_LAW_34516/1f93873ba975a49a6750550208632ea9ec2bc676/
```

Comment:
This is a corporate website with copyrighted content. As such, the list isn't MIT, so it can't be published in Faker.

Reply:
Oh, sorry. Can I use link to Wikipedia then?

Reply:
Sure, if the data is from wikipedia, too.